src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    <Article title={article.title} />
```

Comment:
You need just one Article component here, and it should accept all 3 props at the same time.